models/producer.js

```diff
@@ -6,20 +6,19 @@ const Schema = mongoose.Schema;
 // create new instance of the mongoose.schema. the schema takes an object that shows
 // the shape of your database entries.
 const ProducersSchema = new Schema({
-    firstName: { type: String, required: true },
-    lastName: { type: String, required: true },
-    phoneNumber: { type: String, required: true },
-    country: { type: String, required: true },
-    state: { type: String, required: true },
-    localGovernment: { type: String, required: true },
-    address: { type: String, required: true },
-    typeOfProducts: { type: Array },
+    address: { required: true, type: String },
+    country: { required: true, type: String },
     createdAt: { type: Date },
+    deletedAt: { type: Date },
+    firstName: { required: true, type: String },
+    lastName: { required: true, type: String },
+    localGovernment: { required: true, type: String },
+    phoneNumber: { required: true, type: String },
+    state: { required: true, type: String },
+    typeOfProducts: { type: Array },
```

Comment:
wow, am seeing alphabetical ordering all the place. this is nice

Reply:
Yup, it is now enforced as a linting rule